src/layout/Header/MobileMenu/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react'
+import {
+  Frame,
+  LightModeIcon,
+  NightModeIcon,
+  TransactionButton,
+  TransactionIcon,
+  Items,
+  Line,
+  ThemeMode
+} from './styled'
+import { MenuLinks } from '../../Sidebar/MenuLinks'
+import SocialMedia from 'src/layout/SocialMedia'
+import { Box, Text } from '@pangolindex/components'
+import { Scrollbars } from 'react-custom-scrollbars'
+import Logout from 'src/assets/svg/menu/logout.svg'
+import { useDarkModeManager } from 'src/state/user/hooks'
+import LanguageSelection from 'src/components/LanguageSelection'
+import { MobileHeader } from '../MobileHeader'
+import Footer from 'src/layout/Footer'
+import { useTranslation } from 'react-i18next'
+import MobileWeb3Status from './MobileWeb3Status'
+import TransactionModal from './TransactionModal'
+import Modal from 'src/components/Beta/Modal'
+
+interface Props {
+  activeMobileMenu: boolean
+  handleMobileMenu: () => void
+}
+
+const MobileMenu: React.FC<Props> = ({ activeMobileMenu, handleMobileMenu }) => {
+  const [isDark, toggleDarkMode] = useDarkModeManager()
+
+  const [openTransactions, setOpenTransactions] = useState(false)
+
+  const { t } = useTranslation()
+
+  const onCloseTransactions = () => {
+    setOpenTransactions(false)
+  }
+
+  return (
+    <Frame>
+      <Scrollbars>
+        <MobileHeader activeMobileMenu={activeMobileMenu} handleMobileMenu={handleMobileMenu} />
+        <Box display="flex" flexDirection="column" padding="15px">
+          <Line />
+          <Items>
+            <Box width="100%">
+              <MobileWeb3Status />
+            </Box>
+            <Box display="flex" style={{ gap: '20px' }}>
+              <TransactionButton onClick={() => setOpenTransactions(true)}>
+                <Box width="20px" height="20px">
+                  <TransactionIcon />
+                </Box>
+              </TransactionButton>
+              <LanguageSelection isBeta={true} />
+              <ThemeMode onClick={() => toggleDarkMode()}>
+                <Box width="20px" height="20px">
+                  {isDark ? <LightModeIcon /> : <NightModeIcon />}
+                </Box>
+              </ThemeMode>
```

Comment:
there is no space between icons
![2022-07-03 11 38 27](https://user-images.githubusercontent.com/6604146/177027245-5e00311b-a667-475a-aa71-1a9312a6310f.jpg)